### ISSUE TITLE
make apimachinery failing

### ISSUE THREAD

**Issue by chrislovecnm** [kubernetes/kops#1472]
I am getting the following error with apimachinery:

```
$ make apimachinery
# conversion: go install ./cmd/libs/go2idl/conversion-gen
/Users/clove/Workspace/bin/conversion-gen --skip-unsafe=true --input-dirs k8s.io/kops/pkg/apis/kops/v1alpha1 --v=8  --output-file-base=zz_generated.conversion
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x58 pc=0x5df7f]

goroutine 1 [running]:
panic(0x295cc0, 0xc420012100)
	/usr/local/Cellar/go/1.7/libexec/src/runtime/panic.go:500 +0x1a1
k8s.io/kubernetes/cmd/libs/go2idl/conversion-gen/generators.getManualConversionFunctions(0xc429573b00, 0x0, 0xc4235885d0)
	/Users/clove/Workspace/src/k8s.io/kubernetes/cmd/libs/go2idl/conversion-gen/generators/conversion.go:128 +0x32f
k8s.io/kubernetes/cmd/libs/go2idl/conversion-gen/generators.Packages(0xc429573b00, 0xc4200a2400, 0x2df051, 0x6, 0xc429573b00)
	/Users/clove/Workspace/src/k8s.io/kubernetes/cmd/libs/go2idl/conversion-gen/generators/conversion.go:229 +0x810
k8s.io/kubernetes/vendor/k8s.io/gengo/args.(*GeneratorArgs).Execute(0xc4200a2400, 0xc420018d50, 0x2df051, 0x6, 0x304cb8, 0x2ed2db, 0x5f)
	/Users/clove/Workspace/src/k8s.io/kubernetes/vendor/k8s.io/gengo/args/args.go:165 +0x170
main.main()
	/Users/clove/Workspace/src/k8s.io/kubernetes/cmd/libs/go2idl/conversion-gen/main.go:76 +0x27d
make: *** [apimachinery] Error 2
```

@justinsb what version are you running, or who owns it in k8s core?

**Comment by chrislovecnm** [comment]
@heschlie is getting

```
heschlie@Dalaran:kops$ make apimachinery 
# conversion: go install ./cmd/libs/go2idl/conversion-gen
/home/heschlie/go_work/bin/conversion-gen --skip-unsafe=true --input-dirs k8s.io/kops/pkg/apis/kops/v1alpha1 --v=8  --output-file-base=zz_generated.conversion
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x58 pc=0x45e3c7]

goroutine 1 [running]:
panic(0x695b00, 0xc420016140)
	/usr/local/go/src/runtime/panic.go:500 +0x1a1
k8s.io/kubernetes/cmd/libs/go2idl/conversion-gen/generators.getManualConversionFunctions(0xc42a04ea80, 0x0, 0xc428e502a0)
	/home/heschlie/go_work/src/k8s.io/kubernetes/cmd/libs/go2idl/conversion-gen/generators/conversion.go:128 +0x1d7
k8s.io/kubernetes/cmd/libs/go2idl/conversion-gen/generators.Packages(0xc42a04ea80, 0xc4200ae400, 0x6deed6, 0x6, 0xc42a04ea80)
	/home/heschlie/go_work/src/k8s.io/kubernetes/cmd/libs/go2idl/conversion-gen/generators/conversion.go:228 +0x810
k8s.io/kubernetes/vendor/k8s.io/gengo/args.(*GeneratorArgs).Execute(0xc4200ae400, 0xc420011200, 0x6deed6, 0x6, 0x704f10, 0x6ed512, 0x5f)
	/home/heschlie/go_work/src/k8s.io/kubernetes/vendor/k8s.io/gengo/args/args.go:165 +0x170
main.main()
	/home/heschlie/go_work/src/k8s.io/kubernetes/cmd/libs/go2idl/conversion-gen/main.go:76 +0x27d
Makefile:296: recipe for target 'apimachinery' failed
make: *** [apimachinery] Error 2
```

@heschlie which version of go, and what os?

**Comment by chrislovecnm** [comment]
I am guessing this broke it https://github.com/kubernetes/kubernetes/commit/6a4d5cd7cc58e28c20ca133dab7b0e9e56192fe3

**Comment by heschlie** [comment]
Ubuntu 16.04 Kernel 4.8.0
Go 1.7.1

**Comment by chrislovecnm** [comment]
@justinsb wondering if we should see if a tagged version functions the same

**Comment by chrislovecnm** [comment]
@deads2k what is the best repo to file an issue about this? I noticed that we have a new API Machinery repo

**Comment by kris-nova** [comment]
Closed with #1481
